src/client/.env

```diff
@@ -1,4 +1,5 @@
 VITE_VERSION=$npm_package_version
 VITE_NAME=$npm_package_name
 REACT_EDITOR=code
-EXTEND_ESLINT=true
\ No newline at end of file
+EXTEND_ESLINT=true
+URL_PATH=http://localhost:1917/
```

Comment:
I would not expect a localhost address to be in the general .env file (perhaps `.env.development` ?)